lib/server/serverJquery.js

```diff
@@ -1,8 +1,11 @@
 "use strict";
 
-var jsdom = require("jsdom").jsdom;
+var serverWindow = require("jsdom").jsdom().parentWindow;
 
-module.exports = require("jquery")(jsdom().parentWindow);
+serverWindow.HTMLFrameElement._init = function() {};
```

Comment:
the [body of the _init function](https://github.com/tmpvar/jsdom/blob/3.x/lib/jsdom/level2/html.js#L2042) sets up the relationship between the singleton window object on the server and the iframe window created for each frame element.